.helm/ecamp3/templates/frontend_configmap.yaml

```diff
@@ -21,6 +21,8 @@ data:
       VERSION: '{{ .Values.deployedVersion }}',
       VERSION_LINK_TEMPLATE: '{{ .Values.versionLinkTemplate }}',
       TERMS_OF_SERVICE_LINK_TEMPLATE: '{{ .Values.termsOfServiceLinkTemplate }}',
+      NEWS_LINK: '{{ .Values.newsLink }}',
+      HELP_LINK: '{{ .Values.helpLink }}',
```

Comment:
If the value may not be available, you need to wrap it in an if like the other optional values
Each individually

Reply:
The termsOfServiceLinkTemplate has the same problem, and @manuelmeister made it consistent.
I opened another issue to fix this: https://github.com/ecamp/ecamp3/issues/4777